ui/app/rif/pages/domainsDetailPage/domainDetailActive/domainDetailActive.js

```diff
@@ -0,0 +1,197 @@
+			return <div key={index} className={'resolver-network-description'}>
+					<FontAwesomeIcon icon={getIconForToken(network.networkIcon).icon} color={getIconForToken(network.networkIcon).color} className={'domain-icon'}/>
+					<span>{network.networkName}</span>
+					<span className={'resolver-network-description-address'}>{network.address}</span>
+				</div>
+			})
+		return (
+		<div className={'body'}>
+            <div id='headerName' className={'domain-name'}>
+                <span>{domainName}</span>
+                {isOwner &&
+                    <FontAwesomeIcon icon={faCheckCircle} color="#000080" className={'domain-icon'}/>
+                }
+                {isLuminoNode &&
+                    <FontAwesomeIcon icon={faBolt} color="#508871" className={'domain-icon'}/>
+                }
+                {isRifStorage &&
+                    <FontAwesomeIcon icon={faArchive} color="#AD3232" className={'domain-icon'}/>
+                }
+            </div>
+            <div id='domainDetailBody' className={'domain-detail-body'}>
+                <div id='bodyDescription' className={'domain-description'}>
+                    <div><span className={'domain-description-field'}>Address:&nbsp;</span><span className={'domain-description-value'}>{address}</span></div>
+                    <div><span className={'domain-description-field'}>Content:&nbsp;</span><span className={'domain-description-value'}>{content}</span></div>
+                    <div><span className={'domain-description-field'}>Expires on:&nbsp;</span><span className={'domain-description-value'}>{expirationDate}</span></div>
+                    <div><span className={'domain-description-field'}>Auto renew: <a href={this.props.setAutoRenew()}>{autoRenew ? "on" : "off"}</a></span></div>
+                    <div><span className={'domain-description-field'}>Owner:&nbsp;</span><span className={'domain-description-value'}>{ownerAddress}</span></div>
+                </div>
+                {isOwner &&
+                    <div id='resolversBody' className={'resolvers-body'}>
+                        <div className='resolver-body-top'>
+                            <div id='selectResolver' className={'custom-select'}>
+                                <select id='comboResolvers' className="select-css" onChange={(value) => this.setState({selectedResolverIndex:value.target.selectedIndex})}>
+                                    {this.state.resolvers.map((resolver, index) => {
+                                            return <option key={index} value={resolver.name}>{resolver.name}</option>
+                                        })
+                                    }								
+                                </select>
+                            </div>
+                            <div id='buttonNew' className={'custom-select'}>
+                                <CustomButton 
```

Comment:
This component is used in more than one screen right? What about generalize it?

Reply:
I removed the div tag, cause it was not needed now.